build-tests/api-documenter-test/etc/yaml/api-documenter-test/docbaseclass.yml

```diff
@@ -1,17 +1,17 @@
 ### YamlMime:UniversalReference
 items:
-  - uid: api-documenter-test.DocBaseClass
+  - uid: 'api-documenter-test!DocBaseClass:class'
     summary: Example base class
     name: DocBaseClass
     fullName: DocBaseClass
     langs:
       - typeScript
     type: class
-    package: api-documenter-test
+    package: api-documenter-test!

```

Comment:
I'm assuming the package `uid` goes here and not the package name.  Could someone confirm that?  (I don't have easy access to a DocFX setup, so I wasn't able to test this change myself.)

Reply:
I'm not sure `package` is actually used in DocFX today, other than `type: package`. I can't find any reference to the property in the DocFX sources or templates.

Reply:
From what I've seen in the DocFX test files, `package:` seems to be the uid.